pyproject.toml

```diff
@@ -22,6 +22,7 @@ pre-commit = "^2.20.0"
 coverage = "^7.1.0"
 pytest-profiling = "^1.7.0"
 pytest-benchmark = "^4.0.0"
+mdanalysis = "^2"
```

Comment:
should this be an optional dependency?

Reply:
Its is in the `[tool.poetry.group.dev.dependencies]`